[2024-09-10T01:37:58.825+0800] {processor.py:186} INFO - Started process (PID=58000) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:37:58.826+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T01:37:58.826+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:37:58.826+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:37:58.840+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:37:58.839+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T01:37:58.840+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:37:58.840+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T01:37:58.840+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:37:58.871+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2024-09-10T01:39:48.529+0800] {processor.py:186} INFO - Started process (PID=58101) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:39:48.530+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T01:39:48.530+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:39:48.530+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:39:48.544+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:39:48.544+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T01:39:48.545+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:39:48.544+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T01:39:48.545+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:39:48.577+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-09-10T01:42:29.599+0800] {processor.py:186} INFO - Started process (PID=58244) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:42:29.600+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T01:42:29.600+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:42:29.600+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:42:29.615+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:42:29.615+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T01:42:29.615+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:42:29.615+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T01:42:29.615+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:42:29.649+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-09-10T01:59:32.760+0800] {processor.py:186} INFO - Started process (PID=67508) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:59:32.761+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T01:59:32.761+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:59:32.761+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:59:32.775+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:59:32.774+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T01:59:32.775+0800] {logging_mixin.py:190} INFO - [2024-09-10T01:59:32.775+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T01:59:32.775+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T01:59:32.808+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-09-10T02:01:54.778+0800] {processor.py:186} INFO - Started process (PID=67931) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:01:54.778+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T02:01:54.779+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:01:54.779+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:01:54.793+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:01:54.793+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T02:01:54.793+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:01:54.793+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T02:01:54.793+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:01:54.825+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-09-10T02:04:39.523+0800] {processor.py:186} INFO - Started process (PID=68123) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:04:39.524+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T02:04:39.524+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:04:39.524+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:04:39.538+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:04:39.538+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T02:04:39.539+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:04:39.539+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T02:04:39.539+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:04:39.574+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-09-10T02:07:38.159+0800] {processor.py:186} INFO - Started process (PID=68426) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:07:38.159+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T02:07:38.160+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:07:38.160+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:07:38.175+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:07:38.174+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T02:07:38.175+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:07:38.175+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T02:07:38.175+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:07:38.209+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-09-10T02:13:03.960+0800] {processor.py:186} INFO - Started process (PID=69289) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:13:03.960+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T02:13:03.961+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:13:03.961+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:13:03.975+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:13:03.975+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T02:13:03.976+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:13:03.975+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T02:13:03.976+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:13:04.008+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2024-09-10T02:16:26.394+0800] {processor.py:186} INFO - Started process (PID=69746) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:16:26.394+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T02:16:26.395+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:16:26.395+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:16:26.409+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:16:26.409+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T02:16:26.409+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:16:26.409+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T02:16:26.409+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:16:26.441+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-09-10T02:18:21.366+0800] {processor.py:186} INFO - Started process (PID=69902) to work on /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:18:21.366+0800] {processor.py:914} INFO - Processing file /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-09-10T02:18:21.367+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:18:21.367+0800] {dagbag.py:587} INFO - Filling up the DagBag from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:18:21.382+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:18:21.382+0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-09-10T02:18:21.382+0800] {logging_mixin.py:190} INFO - [2024-09-10T02:18:21.382+0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-09-10T02:18:21.382+0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-09-10T02:18:21.415+0800] {processor.py:208} INFO - Processing /Users/erniesg/.pyenv/versions/3.12.5/envs/letters_home/lib/python3.12/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
